rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -734,6 +744,64 @@ void getOldFederationBtcUTXOs_calledTwice_returnCachedUtxos() {
 
     }
 
+    @Test
+    void getFederationElection_whenElectionIsInStorage_shouldReturnNewElection() {
+
+        // Arrange
+
+        AddressBasedAuthorizer authorizer = getTestingAddressBasedAuthorizer();
```

Comment:
What do you think of implementing this for fed change callers? https://github.com/rsksmart/rskj/blob/master/rskj-core/src/test/java/co/rsk/peg/feeperkb/FeePerKbVoteCaller.java

It will probably be useful in voteFedChange tests

Reply:
Updated.